SpatialGDK/Source/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -139,6 +139,10 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 void USpatialNetDriver::Connect()
 {
 	Connection->OnConnected.BindUFunction(this, FName("OnConnected"));
+	Connection->OnConnectFailed.BindLambda([this](FString Reason)
```

Comment:
Super nit: could we make both of these registrations look the same? As in we either bind UFunctions for both or bind lambdas for both?